### PR TITLE
fix: invalid sp in the sector list page

### PR DESCRIPTION
**Pull request by strahe**
image

image

### CODE REVIEW

**Comment by LexLuthr** [comment]
@strahe Make sure to get screenshots of this table. Especially the right most part. Since we are replacing a column, I don't think render should be affected but I would like to be sure. Also, this change will break `terminateSectors` method as the input for that method will change. You need do the following:
1. Update the index.html to use the updated column name "MinerAddress" as part of terminate &  delete
2. Update terminateSectors to set address string as input and handle from there

**Comment by strahe** [comment]
image

image

File: web/api/sector/sector.go [hunk 1]
```diff
@@ -96,7 +98,7 @@ func (c *cfg) getSectors(w http.ResponseWriter, r *http.Request) {
 	// TODO get sector info from chain and from database, then fold them together
 	// and return the result.
 	type sector struct {
-		MinerID        int64 `db:"miner_id"`
+		MinerID        int64 `db:"miner_id" json:"-"`
```
Comment (LexLuthr) [inline web/api/sector/sector.go:101]: Why do we need this json key?

Comment (strahe) [reply web/api/sector/sector.go:101]: Because the frontend part no longer requires MinerID, but the Go part still does, it is hidden.

Comment (LexLuthr) [reply web/api/sector/sector.go:101]: Okay. We should have been okay even if we sent this key to frontend.

Comment (strahe) [reply web/api/sector/sector.go:101]: Should I remove this json tag?

Comment (LexLuthr) [reply web/api/sector/sector.go:101]: Let's remove it for now. We might end up needing that ID later.

**Review by LexLuthr — APPROVED** [review]

**Pull request merged**